src/vector/index.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="en" style="height: 100%;">
+<html id="root" lang="en" style="height: 100%;">
```

Comment:
why adding a redundant id instead of using `document.getElementsByTagName` ?

Reply:
No reason other than I thought that might be more explicit.  I wasn't sure which would be more ideal.  I can go ahead and change this to use `document.getElementsByTagName` instead if that is preferred.

Reply:
There's a chance that `home.html`, when opened, contains extra `html` tags. To prevent conflicts, assigning an id is probably best. Although I'd prefix it to avoid more conflicts (eg `mx_PageRoot`).